### PR TITLE
Fix front page

### PR DESCRIPTION
**Pull request by saiyan86**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

It fixes the front page of shifu.run, which is SUPER IMPORTANT.

### CODE REVIEW

**Comment by saiyan86** [comment]
@rachelzhang0922 please take a look. Thanks!

File: docusaurus.config.js [hunk 1]
```diff
@@ -213,6 +214,7 @@ const config = {
       hotjar: {
         applicationId: 3065662
       },
+      metadata: [{name: 'keywords', content: 'cloud-native IoT platform, open-source IoT development framwork'}],
```
Comment (kris21he) [inline docusaurus.config.js:217]: framwork -> framework

File: i18n/zh-Hans/code.json [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
-  "Automated collection of protocol and driver's value": {
-    "message": "无论是协议规定的数据，还是驱动返回值，Shifu都能自动采集"
+  "Automatically capture data from various protocols and drivers": {
+    "message": "自动采集各类协议和驱动的数据"
   },
   "Open Source Community": {
     "message": "开源社区"
   },
-  "Open source with limitless cloud carrier choice Versatile deployment on enterprise, edge, private or public cloud service": {
+  "Shifu is open source and no vendor lock-in. Support on-prem, edge, private, hybrid and public cloud deployment": {
```
Comment (rachelzhang0922) [inline i18n/zh-Hans/code.json:26]: on-premise sounds more official

File: i18n/zh-Hans/code.json [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
   "Infinite Scalability": {
     "message": "无限扩容"
   },
-  "Billions level of device service scalability Big enough to run the city": {
+  "The kubernetes-based architecture allows Shifu to expand infinitely, easily handle billions of devices integration, even for smart city projects": {
     "message": "基于k8s的架构让Shifu可以无限扩容，轻松处理亿级设备接入，连智慧城市项目都不在话下"
   },
-  "Orchestration for IoT Application Development": {
-    "message": "让开发产业场景像开发APP一样简单！"
+  "Kubernetes native reliability for IoT application development as easy as developing web apps": {
```
Comment (rachelzhang0922) [inline i18n/zh-Hans/code.json:53]: this sentence seems ungrammatical

 to render IoT application development as easy as developing web apps

File: i18n/zh-Hans/code.json [hunk 4]
```diff
@@ -62,7 +62,7 @@
   "With private or public protocol": {
     "message": "小到使用公有协议的温湿度计"
   },
-  "Shifu's high compatibility design makes it easy for you to handle universal devices": {
+  "Shifu's high compatibility allows you to easily deal with various heterogeneous devices": {
```
Comment (rachelzhang0922) [inline i18n/zh-Hans/code.json:65]: I'd prefer 'handle' here

File: src/components/Homepage/introduction/index.js [hunk 5]
```diff
@@ -41,24 +41,24 @@ function IntroBar(props) {
...
   {
     img: require('@site/static/img/index/guide-access.svg').default,
-    title: translate({ message: 'Extremely Fast Access' }),
+    title: translate({ message: 'Extremely Fast Integration' }),
     descript:
       <span>
-        <Translate>From machinery to thermometers</Translate><br />
-        <Translate>With private or public protocol</Translate><br />
-        <Translate>Shifu's high compatibility design makes it easy for you to handle universal devices</Translate>
+        <Translate>Robots to thermometers</Translate><br />
```
Comment (rachelzhang0922) [inline src/components/Homepage/introduction/index.js:47]: From robots to thermometers

File: i18n/zh-Hans/code.json [hunk 6]
```diff
@@ -1,59 +1,59 @@
...
   },
-  "Compatible with all K8s-based cloud/edge frameworks (OpenYurt, KubeEdge, etc.)": {
+  "Compatible with all Kubernetes-based cloud/edge frameworks (OpenYurt, KubeEdge, etc.)": {
     "message": "Shifu兼容所有基于k8s的云边协同框架（OpenYurt, KubeEdge等）"
   },
   "Infinite Scalability": {
     "message": "无限扩容"
   },
-  "Billions level of device service scalability Big enough to run the city": {
+  "The kubernetes-based architecture allows Shifu to expand infinitely, easily handle billions of devices integration, even for smart city projects": {
```
Comment (tomqin93) [inline i18n/zh-Hans/code.json:50]: need to capitalize K in Kubernetes

**Review by tomqin93 — COMMENTED** [review]
added 1 comment

**Comment by Yang-Xijie** [comment]
After modifying the en text, you should run `yarn write-translations --locale zh-Hans` to re-generate `code.json`, and translate new items.

For more details, check <https://github.com/Edgenesis/shifu.run/blob/main/README.md#internationalization>.

File: i18n/zh-Hans/code.json (hunk 6 above)
Comment (rachelzhang0922) [reply i18n/zh-Hans/code.json:50]: this line is outdated

File: i18n/zh-Hans/code.json (hunk 3 above)
Comment (rachelzhang0922) [reply i18n/zh-Hans/code.json:53]: this line is outdated

**Comment by tomqin93** [comment]
@rachelzhang0922 please sync with main to address conflicts

**Review by tomqin93 — APPROVED** [review]

**Pull request merged**